EIPS/eip-5094.md

```diff
@@ -0,0 +1,105 @@
+---
+eip: 5094
+title: URI Format for Ethereum Network Switching
+author: Luc van Kampen (@lucemans), Jakob Helgesson (@svemat01), Joshua Hendrix (@thejoshuahendrix)
+type: Standards Track
+category: ERC
+status: Draft
+created: 2022-05-13
+requires: 681, 831
+---
+
+## Simple Summary
+
+A standard way of representing various network configurations as URLs.
+
```

Comment:
```suggestion
```
Simple Summary is no longer a valid section. See https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md for current template.